yapapi/rest/activity.py

```diff
@@ -44,7 +44,7 @@ async def new_activity(self, agreement_id: str) -> "Activity":
             activity_id = await self._api.create_activity(agreement_id)
             return Activity(self._api, self._state, activity_id)
         except yexc.ApiException:
-            _log.error("Failed to create activity for agreement %s", agreement_id)
+            _log.debug("Failed to create activity for agreement %s", agreement_id, exc_info=True)
```

Comment:
No need to print this message to the console, it belongs to the debug log (in exchange, we add exception information).